apps/prairielearn/src/middlewares/authzCourseOrInstance.js

```diff
@@ -203,118 +198,108 @@ module.exports = function (req, res, next) {
 
       async.series(
         [
-          (callback) => {
+          async () => {
```

Comment:
If these are all `async` now, could we get rid of the `async.series`?

Reply:
If we were so inclined, yeah, though we use the separate functions for control flow in a way that is sliiiiiightly more annoying to refactor. I can come back and do that later, but I'll skip over that for now.